src/libstd/error.rs

```diff
@@ -212,6 +212,13 @@ impl<T: Error> Error for Box<T> {
     }
 }
 
+#[unstable(feature = "fmt_error", issue = "33827")]
```

Comment:
Unfortunately we don't actually read stability attributes on `impl` blocks for traits, so this'll be "insta-stable". It's fine to just change this to `#[stable]`